src/rpc/handlers/AccountLines.cpp

```diff
@@ -102,6 +102,12 @@ AccountLinesHandler::addLine(
     line.qualityIn = lineQualityIn;
     line.qualityOut = lineQualityOut;
 
+    if (lineNoRipple)
```

Comment:
Is the intention to check that the optional is not `nullopt` and unconditionally set `noRipple` to true if it is? This code does not consider nor set it to the actual value inside of the optional - just wanted to check that it is the intention.